build/cqs.tsv

```diff
@@ -116,6 +117,8 @@ javax.measure:unit-api	1.0	compile
 javax.xml.bind:jaxb-api	2.2.7	compile
 joda-time:joda-time	2.3	compile
 log4j:log4j	1.2.17	compile
+net.java.dev.jna:jna	4.5.2	compile
```

Comment:
4.5.1 is approved... is that close enough? https://dev.eclipse.org/ipzilla/show_bug.cgi?id=15644

Reply:
https://clearlydefined.io/definitions/maven/mavencentral/net.java.dev.jna/jna/4.5.2